.circleci/config.yml

```diff
@@ -0,0 +1,173 @@
+version: 2
+
+jobs:
+
+  test:
+    docker:
+      - image: rust:latest
+    steps:
+      - checkout
+      # Load cargo target from cache if possible.
+      # Multiple caches are used to increase the chance of a cache hit.
+      - restore_cache:
+          keys:
+            - v1-cargo-cache-test-{{ arch }}-{{ .Branch }}
+            - v1-cargo-cache-test-{{ arch }}
+
+      # Install nightly & wasm
+      - run:
+          name: Install Rust nightly
+          command: rustup update nightly && rustup default nightly
+      - run:
+          name: Add wasm32 target
+          command: rustup target add wasm32-unknown-unknown
+
+      # Install wasm tools
+      - run:
```

Comment:
We actually don't use `wasm-bindgen-cli` anymore since we're using `wasm-pack`. I just never removed it from the `.travis.yml`, so this can be removed!